component-server-parent/component-server/src/main/java/org/talend/sdk/component/server/front/ComponentResourceImpl.java

```diff
@@ -570,36 +574,36 @@ private ComponentIndex toComponentIndex(final Container container, final Locale
         final String familyIcon = meta.getParent().getIcon();
         final IconResolver.Icon iconContent = iconResolver.resolve(container, icon);
         final IconResolver.Icon iconFamilyContent = iconResolver.resolve(container, familyIcon);
-        final String familyDisplayName =
-                meta.getParent().findBundle(loader, locale).displayName().orElse(meta.getParent().getName());
+        final FamilyBundle parentBundle = meta.getParent().findBundle(loader, locale);
+        final ComponentBundle bundle = meta.findBundle(loader, locale);
+        final String familyDisplayName = parentBundle.displayName().orElse(meta.getParent().getName());
         final List<String> categories = ofNullable(meta.getParent().getCategories())
                 .map(vals -> vals
                         .stream()
                         .map(this::normalizeCategory)
-                        .map(category -> category.replace("${family}", meta.getParent().getName())) // not
-                        // i18n-ed
-                        // yet
-                        .map(category -> meta
-                                .getParent()
-                                .findBundle(loader, locale)
-                                .category(category)
-                                .orElseGet(() -> category
-                                        .replace("/" + meta.getParent().getName() + "/",
-                                                "/" + familyDisplayName + "/")))
+                        .map(category -> category.replace("${family}", meta.getParent().getName()))
+                        // not i18n-ed yet
```

Comment:
I think this comment should be above the line 584
.map(category -> category.replace("${family}", meta.getParent().getName()))
it was related to it.
Now it looks like it comments the 586

Reply:
right! will simply remove comment, it's useless.